### PR TITLE
LRQA-77264 Remove workflow instances after test

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by naraandrade** [comment]
ci:test:relevant

File: modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/workflow/handler/test/LayoutWorkflowHandlerTest.java [hunk 1]
```diff
@@ -108,6 +111,17 @@ public void setUp() throws Exception {
 
 	@After
 	public void tearDown() throws Exception {
+		List<WorkflowInstance> workflowInstances =
+			WorkflowInstanceManagerUtil.getWorkflowInstances(
+				TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
+				new String[] {Layout.class.getName()}, false, -1, -1, null);
```
Comment (MarinhoFeliphe) [inline modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/workflow/handler/test/LayoutWorkflowHandlerTest.java:117]: Hi @seltong,

just replace `-1, -1` to `QueryUtil.ALL_POS, QueryUtil.ALL_POS`

Thank you!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
Hi @seltong,

feel free to forward it to Bryan

Thank you!

**Comment by seltong** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/124152

**Pull request closed without merging**